contracts/REXLimitOrderManager.sol

```diff
@@ -0,0 +1,123 @@
+pragma solidity ^0.8.0;
+
+import {ISuperfluid} from "@superfluid-finance/ethereum-contracts/contracts/interfaces/superfluid/ISuperfluid.sol";
+
+import {ISuperToken} from "@superfluid-finance/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol";
+
+import {SuperTokenV1Library} from "@superfluid-finance/ethereum-contracts/contracts/apps/SuperTokenV1Library.sol";
+
+import "./IREXUniswapV3Market.sol";
+
+/**
+ * @title Ricochet's Limit Order Manager
+ *
+ * @notice This contract manages limit orders in the Ricochet protocol.
+ *
+ */
+contract REXLimitOrderManager {
```

Comment:
Could this also be a gelato resolver? Add a `checker()` method and inherit from GelatoResovler. I didn't see the resolver yet and thought about it.